config.toml

```diff
@@ -171,13 +172,13 @@ themesDir = "node_modules/"
   pre = "<i data-feather=\"at-sign\"></i>"
   weight = 3
   
-[[menu.sidebar]]
-  parent ="community"
-  name = "Resources"
-  url = "/community/resources"
-  pre = "<i data-feather=\"folder\"></i>"
-  weight = 4
-  
+# [[menu.sidebar]]
+#   parent ="community"
+#   name = "Resources"
+#   url = "/community/resources"
+#   pre = "<i data-feather=\"folder\"></i>"
+#   weight = 4
+
```

Comment:
For the changes here,
I was confused by the previous logic that we need to create middle-layer sidebar menus twice in order to show on the page so that I made some modifications to just create once. Please also refer to the changes I made in "layouts/partials/sidebar.html".

This gives me a better understanding to create the mobile-sidebar.